index.html

```diff
@@ -650,21 +654,37 @@ <h3>The <dfn data-dfn-for="GlobalEventHandlers" data-dfn-type="event">lostpointe
+                <section>
+                  <h4>Attributes (other than coordinates)</h4>
+                  <p>For these events, all <code>PointerEvent</code> specific attributes (defined in this spec) other than <code>pointerId</code> and <code>pointerType</code> MUST have their default values. In addition:</p>
+                  <ul>
+                    <li>If the events are generated by a pointing device, their <code>pointerId</code> and <code>pointerType</code> MUST be the same as the PointerEvents that caused these events.</li>
+                    <li>If the events are generated by a non-pointing device (such as voice recognition software or a keyboard interaction), <code>pointerId</code> MUST be <code>-1</code> and <code>pointerType</code> MUST be an empty string.</li>
+                  </ul>
+                </section>
+                <section>
+                  <h4>Event coordinates</h4>
+                  <p>As noted in {{PointerEvent}}, [[[CSSOM-VIEW]]] proposes to redefine the various coordinate properties (<code>screenX</code>, <code>screenY</code>, <code>pageX</code>, <code>pageY</code>, <code>clientX</code>,
+                      <code>clientY</code>, <code>x</code>, <code>y</code>, <code>offsetX</code>, <code>offsetY</code>) as <code>double</code>, to allow for fractional coordinates.
+                      However, this change — when applied only to {{PointerEvent}}, but not to regular {{MouseEvent}} — has proven to lead to web compatibility issues with legacy code
+                      in the case of <code>click</code>, <code>auxclick</code>, and <code>contextmenu</code>. For this reason, user agents that have implemented the proposed
+                      change in [[[CSSOM-VIEW]]] only for {{PointerEvent}} MUST convert the various coordinate properties for the <code>click</code>, <code>auxclick</code>, and <code>contextmenu</code>
+                      to <code>long</code> values (as defined in the original [[[UIEVENTS]]]) using <a data-cite="ECMASCRIPT#sec-math.floor">Math.floor</a> [[ECMASCRIPT]].</p>
+                </section>
+                <section>
+                  <h4>Event dispatch</h4>
+                  <p>If a <code>click</code>, <code>auxclick</code> or <code>contextmenu</code> event <var>E</var> is fired in response to a {{GlobalEventHandlers/pointerup}} event, and
+                    the {{GlobalEventHandlers/pointerup}} event was disaptched while the pointer was captured, <var>E</var> MUST be dispatched to the target of the {{GlobalEventHandlers/pointerup}} event.
+                    Otherwise, <var>E</var> MUST follow the dispatch process defined in [[UIEVENTS]].</p>
+                  <p>For the special case that a <code>contextmenu</code> event <var>E</var> is fired in response to a {{GlobalEventHandlers/pointerdown}} event (as happens on certain platforms for legacy reasons), and
```

Comment:
It is not a special case. It is just platform convention. On Linux/Gnome for example, contextmenu is triggered on down event.

Reply:
Right, triggering at up is less common.  Rephrased, ptal.